src/main/scala/cromwell/engine/db/slick/DatabaseConfig.scala

```diff
@@ -0,0 +1,15 @@
+package cromwell.engine.db.slick
+
+import com.typesafe.config.ConfigFactory
+import slick.util.ConfigExtensionMethods._
+
+object DatabaseConfig {
+  private val config = ConfigFactory.load()
+  lazy val rootDatabaseConfig = config.getConfig("database")
+  private val databaseConfigName = rootDatabaseConfig.getStringOr("config", null)
```

Comment:
Could we make this an `Option` immediately so there are no `null` checks hanging about?